src/mpid/common/shm/mpidu_shm.h

```diff
@@ -0,0 +1,58 @@
+/* -*- Mode: C; c-basic-offset:4 ; indent-tabs-mode:nil ; -*- */
+/* vim: set ft=c.mpich : */
+/*
+ *  (C) 2019 by Argonne National Laboratory.
+ *      See COPYRIGHT in top-level directory.
+ */
+
+#ifndef MPIDU_SHM_H_INCLUDED
+#define MPIDU_SHM_H_INCLUDED
+
+size_t MPIDU_shm_get_mapsize(size_t size, size_t * psz);
+
```

Comment:
Can you please add comment for `MPIDU_shm_get_mapsize`? It is not straightforward that `psz` is a `inout`. And it might be better if we rename `psz` to `page_sz`, and rename `size` to `len` (to be consistent with the other ` MPIDU_shm_` calls)